### ISSUE TITLE
Thick insertion

### ISSUE THREAD

**Issue by MagoWyr** [obophenotype/human-phenotype-ontology#9836]
**Preferred term label:**
Thick insertion

**Synonyms**

**Definition (free text, please give PubMed ID)**
For illustration see: WHO laboratory manual for the examination and processing of human semen (6th Edition)

**Parent term (use hpo.jax.org/app)**
HP:0012866

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA: 399808

**Your nano-attribution (ORCID)**
0000-0002-7651-2403

**Comment by pnrobinson** [comment]
@MagoWyr  ideally there is a PMID where we can say See Figure ?? of PMID:123. Alternatively if there is a Figure in the WHO manual we could say see Figure ? in the manual?

We always try to have an English language definition. What is being inserted into what? Is this a kind of Neck and midpiece defects? Our goal is for the definition to be comprehensible to PhD students or MDs from other medical specialties!

**Comment by MagoWyr** [comment]
The figure in the WHO manual is: Fig. 2.10 Schematic drawings of some abnormal forms of human spermatozoa
I will add English language definitions.

**Comment by MagoWyr** [comment]
Definition: 
> 96 % of spermatozoa show increased width of the sperm neck or midpiece.

**Comment by pnrobinson** [comment]
@MagoWyr I do not have access to the WHO book. Could you provide a PMID with a Figure that illustrates this finding and suggest a textual definition? The above definition is "bundled" and does not do more than re-arrange the term label and is not understandable for non-specialists.

**Comment by MagoWyr** [comment]
See the terms Thin neck and Bent neck:

A PMID for this finding with a description is here: PMID: 22992901

For the figure I would use the one in the WHO laboratory manual: [cid:9f1b4978-d01f-4d6b-bf63-0a8cdac56fc8]
________________________________

**Comment by pnrobinson** [comment]
@MagoWyr 
There does not seem to be any good text definition of this abnormality in the WHO article or any other article I can find.
How about this (note: HPO labels need to be unambiguous even outside of the hierarchy -- so just saying thick insertion is not clear enough

Label: Thick spermatozoon midpiece insertion
Definition: Increased width of the midpiece of the spermatozoon. The midpiece is the structure that is located between the head and the flagella (tail) of the spermatozoon. This term applies if the width of the midpiece increases progressively from the flagella to the head and if the proportion of individual spermatozoa with this anomaly is above the upper limit of normal.
Comment: See Figure 2.10i of the WHO laboratory manual for the examination and processing of human semen
Sixth edition (2021).

**Comment by MagoWyr** [comment]
Damit bin ich einverstanden. Danke fuer die Beschreibung!
________________________________

**Comment by pnrobinson** [comment]
@MagoWyr 
added the above as
Thick spermatozoon midpiece insertion
HP:0035008
